### ISSUE TITLE
Question - Object class mapping and number of classes in config

### ISSUE THREAD

**Issue by ChristianIngwersen** [Peterande/D-FINE#99]
Hi,

In the example the following is given 
> self.obj365_ids = [0, 5]  # Person, Cars

is this the ordering that should be used consistently? I.e. the the mapping in `src/data/dataset/coco_dataset.py` -1?

Snippet from file
```
mscoco_category2name = {
    1: 'person',
    2: 'bicycle',
    3: 'car',
    4: 'motorcycle',
    5: 'airplane',
    6: 'bus',
    7: 'train',
    8: 'truck',
    9: 'boat',
    10: 'traffic light',
    11: 'fire hydrant',
    13: 'stop sign',
```

So if I have a custom dataset with e.g. trucks only I should specify:
> self.obj365_ids = [7]  # Truck

And then further in my `custom_detection.yml` config I would specify number of classes as being 2 as I will have two classes background and trucks? Is this correctly understood?

**Comment by Peterande** [comment]
It depends on whether the class id starts at 0 or 1. If you start with 1 you can remap it or just set class to 2 as you said.

**Comment by KhanMechAI** [comment]
Hey team, i've also been having some difficulty finetuning on a custom dataset (using the pre-trained obj365 weights, s model). In the docs it says that you can just use your new labels, but in practice, that doesnt seem to work very well. Should we try to line our classes up with the obj365 or coco classes as best we can? Some clarification or examples would be awesome
